src/GravityVisitor.h

```diff
@@ -95,6 +97,29 @@ class GravityVisitor {
     }
     return false;
   }
+
+  static inline bool intersect(const OrientedBox<Real>& box, Vector3D<Real> pos, Real rsq) {
```

Comment:
Is there somewhere else we can stick this so that it's accessible to all of the Visitor classes? I also need to use this function for the CollisionVisitor.

Reply:
This is similar to code in ../utility/structures/Space.h.  It (or something like it) should go there.

Reply:
I'll go ahead and add this to the utility repository. We need this function in ChaNGa as well.

Reply:
Note that we have a Box-Sphere intersection test.  However, the Sphere class would require a sqrt(), and that is expensive.  Be sure it is very clear that the 3rd argument is r^2, and there is no confusion with similar tests.